## mportant: if the name is not exactly the same as the node name, parameters wonâ€™t be loaded, 
## and ROS2 will not display the error. In this case, your node name (which can be different 
## from the executable name)

/**:
  ros__parameters:
      safe_dist: 0.2
      limit_xi: -2.5          #-1.0
      limit_xs:  2.5              #1.0
      limit_yi:  -2.5             #-1.0
      limit_ys: 2.5               #1.0


/**/poly_gen:
  ros__parameters:
    update_freq: 10.0 # Hz     #2.0
    num_vertices: 6        # 6
    dist_step: 0.05  # m     #0.1
    start_dist: 0.15 #0.15  # m
    obstacle_d_thr: 0.05    # 0.05  # m
    obst_dist_min: 0.1 #m

/**/graph_update:
  ros__parameters:
    node_dist_min: 0.1
    node_dist_open: 0.3
    node_bound_dist: 0.2
    obstacle_margin: 0.05 #0.05


/**/graph_expl:
  ros__parameters:
    distance_exp: 4.0
    reached_tol: 0.1








